corehq/apps/cloudcare/templates/cloudcare/cloudcare_home.html

```diff
@@ -40,7 +40,7 @@
     <script src="{% static 'nprogress/nprogress.js' %}"></script>
     <script src="{% static 'moment/min/moment.min.js' %}"></script>
 
-    <script src="{% static 'hqwebapp/js/ui-element.js' %}"></script>
+    <script src="{% static 'style/js/bootstrap2/ui-element.js' %}"></script>
```

Comment:
it is, but it was still using the old file which got moved to `style/js/bootstrap2/ui-element.js` and I didn't want to switch it over in this PR for fear it might affect something in cloudcare that isn't tested. It's probably fine, but I wanted to keep that change separate